autogen/browser_utils.py

```diff
@@ -26,6 +34,31 @@
 except ModuleNotFoundError:
     pass
 
+# The Selenium package is used to automate web browser interaction from Python
+IS_SELENIUM_CAPABLE = False
+try:
+    from selenium import webdriver
+    from selenium.common.exceptions import TimeoutException
+
+    # from selenium.webdriver.support.ui import WebDriverWait # We might implement this next
+    from selenium.webdriver.common.by import By
+    from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+    from selenium.webdriver.common.keys import Keys
+    from selenium.webdriver.edge.service import Service as EdgeService
+    from selenium.webdriver.edge.options import Options as EdgeOptions
+    from selenium.webdriver.firefox.options import Options as FirefoxOptions
+    from selenium.webdriver.firefox.firefox_profile import FirefoxProfile
+    from selenium.webdriver.chrome.options import Options as ChromeOptions
+
+    IS_SELENIUM_CAPABLE = True
+except ImportError as e:
+    print(f"The module/package '{e.name}' is not available.")
+    print("Try running 'pip install selenium'.  You may need to run 'sudo easy_install selenium' on Linux or MacOS")
+    print(
+        "Official selenium installation documentation: https://www.selenium.dev/documentation/webdriver/getting_started/install_library/"
+    )
+    raise e
```

Comment:
We probably shouldn't print these messges here. Nor raise the error. Only print and raise when selenium is required.

Reply:
Thanks for your feedback, will remove it